modules/services/espanso.nix

```diff
@@ -42,10 +59,31 @@ in
       package = mkOption {
         type = types.package;
         description = "Which espanso package to use";
-        default = pkgs.espanso;
+        default = if isDarwin || cfg.x11Support then pkgs.espanso else pkgs.espanso-wayland;
```

Comment:
Would make more sense, to me, to keep the safe default and show the `espanso-wayland` package as an example alternative.